src/vm/parser.rs

```diff
@@ -56,17 +57,18 @@ fn get_lines_at(input: &str) -> Vec<usize> {
     out
 }
 
-pub fn lex(input: &str) -> Result<Vec<(LexItem, u32)>> {
+pub fn lex(input: &str) -> Result<Vec<(LexItem, u32, u32)>> {
     // Aaron: I'd like these to be static, but that'd require using
     //    lazy_static (or just hand implementing that), and I'm not convinced
     //    it's worth either (1) an extern macro, or (2) the complexity of hand implementing.
 
     let lex_matchers: &[LexMatcher] = &[
         LexMatcher::new(r##""(?P<value>((\\")|([[:print:]&&[^"\n\r\t]]))*)""##, TokenType::StringLiteral),
-        LexMatcher::new(";;[[:print:]&&[^\n\r\t]]*", TokenType::Whitespace), // ;; comments.
+        LexMatcher::new(";;[[:print:]&&[^\n\r]]*", TokenType::Whitespace), // ;; comments.
```

Comment:
good catch!

Reply:
No `\t`?

Reply:
That match case is for comments -- and the inclusion of `\t` in that group would cause `\t`s to end the comment line.

Reply:
Gotcha.